fitbenchmarking/fitbenchmarking
# Adding combined accuracy and runtime tables

#### Description of Work

Fixes #137 

FitBenchmarking now outputs an additional table that has combined accuracy and runtime information.

#### Testing Instructions

1. Run `example_runScripts.py` in the mantidpython console.
2. Check the results folder and the combined table in question.
